src/listener.rs

```diff
@@ -286,8 +286,11 @@ impl Listener {
     }
 
     /// Returns a human-readable string presented in the ngrok dashboard
-    /// and the API. Use the HttpListenerBuilder forwards_to, TcpListenerBuilder forwards_to, etc.
+    /// and the API. Use the :meth:`HttpListenerBuilder.forwards_to`, :meth:`TcpListenerBuilder.forwards_to`, etc.
     /// to set this value explicitly.
+    ///
+    /// To automatically forward connections, you can use :any:`listen_and_forward`,
+    /// or :any:`listen_and_serve` on the Listener Builder. These methods will also set this `forwrads_to` value.
```

Comment:
> forwrads_to

:ohno:

Reply:
thx, fixed in: https://github.com/ngrok/ngrok-python/pull/75